### ISSUE TITLE
To long Challenge interval

### ISSUE THREAD

**Issue by johanneshahn** [EpicCash/epicboxnodejs#4]
The code to renew the challenge/subscribe interval is to long for the epicbox listener (cli wallet)

`setInterval(forIntervalChallenge, 3*60*1000); `

This cause the epicbox listener to loose the connection to epicbox.

The error in epicbox is then:
```
[HDQBZ]  Socket close for  es.....
[9:20:10 AM] Close by code:  1006  and reason  
```

**Comment by who-biz** [comment]
We need to make sure this is actually the reason behind `1006` code. 

I see these same errors and they are often non-fatal and can even be due to Cloudflare (this is when I see them), as `epicbox.blur.cash` is behind cloudflare protection.

**Comment by who-biz** [comment]
Closed by #6
